### PR TITLE
Add banner package

### PR DESCRIPTION
**Pull request by cstrouse**
Banner is an implementation of the traditional Unix-program used to display
large characters.

Tested as working on XE500C13-K01US.

### CODE REVIEW

File: packages/banner.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  depends_on 'shhmsg'
+  depends_on 'shhopt'
+
+  def self.build
+    system 'make'
+  end
+
+  def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
```
Comment (uberhacker) [inline packages/banner.rb]: You might add `PREFIX=#{CREW_PREFIX}` here since I'm assuming there is no configure script available?

Comment (cstrouse) [reply packages/banner.rb]: No configure script but the Makefile defaults to /usr/local. Good point though that it should be explicit.

**Comment by uberhacker** [comment]
According to `man make`, the variable is `prefix` (lowercase).  Not sure if that makes a difference or not but if it's case insensitive, then it would matter.

**Comment by jam7** [comment]
Now sure how their makefile work.
```
$ crew build banner
...
$ tar tvf banner-1.3.2-chromeos-x86_64.tar.xz
-rw-r--r-- chronos/chronos   0 2017-09-05 02:59 dlist
-rw-r--r-- chronos/chronos   0 2017-09-05 02:59 filelist
```
It looks like shhmsg and shhopt also doesn't install binaries through CREW_DEST_DIR.

**Comment by uberhacker** [comment]
@cstrouse: Did you get a chance to look into this issue yet?

**Comment by cstrouse** [comment]
@uberhacker I forgot all about this. I'll see if I can get this resolved today.

**Comment by uberhacker** [comment]
It looks like the dependencies `shhmsg` and `shhopt` both don't abstract the library files but that is not banner's fault.  I'll go ahead and merge this and hopefully, we can get the dependency packages updated eventually.

**Pull request merged**